packages/@o3r/schematics/src/rule-factories/ng-add/schematics-register.ts

```diff
@@ -23,8 +23,51 @@ export function registerPackageCollectionSchematics(packageJson: PackageJson, an
   };
 }
 
+/**
+ * Setup schematics default params in angular.json and per project
+ * @param schematicsDefaultParams default params to setup by schematic
+ * @param projectName The name of the project on which to add the schematics
+ * @param overrideValue Define if the given value should override the one already defined in the workspace
+ */
+export function setupSchematicsParamsForProject(schematicsDefaultParams: Record<string, SchematicOptionObject>, projectName: string | undefined, overrideValue = false): Rule {
```

Comment:
```suggestion
export function setupSchematicsParamsForProject(schematicsDefaultParams: Record<string, SchematicOptionObject>, projectName?: string, overrideValue = false): Rule {
```
so we don't need to specify `undefined` for the `projectName` when it's not needed